x-pack/solutions/security/plugins/elastic_assistant/server/lib/attack_discovery/schedules/types.ts

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { DefaultAlert } from '@kbn/alerts-as-data-utils';
+import { RuleExecutorOptions, RuleType, RuleTypeState } from '@kbn/alerting-plugin/server';
+import {
+  ALERT_ATTACK_DISCOVERY_ALERTS_CONTEXT_COUNT,
+  ALERT_ATTACK_DISCOVERY_ALERT_IDS,
+  ALERT_ATTACK_DISCOVERY_API_CONFIG,
+  ALERT_ATTACK_DISCOVERY_DETAILS_MARKDOWN,
+  ALERT_ATTACK_DISCOVERY_ENTITY_SUMMARY_MARKDOWN,
+  ALERT_ATTACK_DISCOVERY_MITRE_ATTACK_TACTICS,
+  ALERT_ATTACK_DISCOVERY_REPLACEMENTS,
+  ALERT_ATTACK_DISCOVERY_SUMMARY_MARKDOWN,
+  ALERT_ATTACK_DISCOVERY_TITLE,
+  ALERT_ATTACK_DISCOVERY_USERS,
+} from './fields';
+
+export type AttackDiscoveryAlert = DefaultAlert & {
+  [ALERT_ATTACK_DISCOVERY_TITLE]: string;
```

Comment:
nit: Consider ordering the entries in this file alphabetically

Reply:
done in https://github.com/elastic/kibana/pull/215184/commits/05cc112bd7260691402f301864bacdc2cf1d340b and in https://github.com/elastic/kibana/pull/215184/commits/6d61830a29b9d24e0d8bd297bcc50a3350e55835